### PR TITLE
TST: tables + rework fake device instantiation

### PR DESCRIPTION
**Pull request by klauer**
* Fixes tables and gets some coverage on them.
* Fake devices now work much better in testing as values are initialized to non-`None`
* Removed unused code (can add back to beamline configuration if desirable)

### CODE REVIEW

**Comment by klauer** [comment]
Fake device support functions may get merged into ophyd eventually: nsls-ii/ophyd#625

File: jet_tracking/tests/conftest.py [hunk 1]
```diff
@@ -148,6 +171,69 @@ def jet_control(injector, questar, parameters, diffract):
...
+    ns.Offaxis = offaxis_camera
+    ns.OffaxisParams = offaxis_parameters
+    ns.Parameters = parameters
+    ns.PressureController = _instantiate_fake_device(devices.PressureController)
+    ns.Questar = questar
+    ns.Selector = _instantiate_fake_device(devices.Selector)
+    ns.SDS = SDS({})
+    ns.SDS.SDS_devices.extend([ns.Selector, ns.CoolerShaker, ns.HPLC,
+                               ns.PressureController, ns.FlowIntegrator])
+    return ns
```
Comment (ZLLentz) [inline jet_tracking/tests/conftest.py]: I like this pytest fixture pattern, I might steal it

Comment (klauer) [reply jet_tracking/tests/conftest.py]: By all means do! 👍

**Comment by klauer** [comment]
@ZLLentz - that's reasonable, thanks for looking!

**Pull request merged**